config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java

```diff
@@ -41,6 +42,7 @@
         boolean multitenant();
         ApplicationId applicationId();
         List<ConfigServerSpec> configServerSpecs();
+        URI loadBalancerAddress();
```

Comment:
Note @tokle: The `AthenzIdentityProvider` assumes that the load balancer address is a hostname (no scheme or port). We'll have to update that component at the same time of merging this PR.

Reply:
This PR terminates just before building the model. Will add a todo